docs/docsite/rst/installation_guide/intro_installation.rst

```diff
@@ -324,6 +324,33 @@ If you wish to install Ansible globally, run the following commands::
     Please make sure you have the latest version of ``pip`` before installing Ansible.
     If you have an older version of ``pip`` installed, you can upgrade by following `pip's upgrade instructions <https://pip.pypa.io/en/stable/installing/#upgrading-pip>`_ .
 
+Upgrading Ansible from version <= 2.9 to version 2.10 or later
```

Comment:
I think I'd prefer to turn `<=` into words.

Reply:
"Upgrading Ansible from version 2.9 and prior to version 2.10 or later" ?

Reply:
What we use in setup.py is:

> Installing ansible-base with ansible-2.9 or older currently installed with pip is known to cause problems.

Reply:
> What we use in setup.py is:
> 
> > Installing ansible-base with ansible-2.9 or older currently installed with pip is known to cause problems.

The point of view I have is an ansible user that doesn't know about the underlying refactoring and split, 
I wanted to avoid talking about ansible-base.